packages/fortranxml/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fortranxml(AutotoolsPackage):
+    """Fortran XML (FoX) s a library, written entirely in Fortran, designed to allow
+    the easy use of XML from Fortran programs. FoX caters for both XML output
+    and input.
+    """
+
+    homepage = "https://github.com/andreww/fox"
+    git = "https://github.com/andreww/fox.git"
+
+    maintainers = ["andreww"]
```

Comment:
https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers

Reply:
I take it you will be the maintainer, then, @harshula ? :laughing:

Reply:
Sure.

Reply:
Done